docs/kubernetes/ingress-usage.md

```diff
@@ -351,7 +353,7 @@ directly with a HTTP status code 200:
 
 ```
 zalando.org/skipper-routes: |
-  Path("/") -> setResponseHeader("X", "bar") -> inlineContent("<html><body>hello</body></html>") -> status(200) -> <shunt>
+  PathRegexp("/") -> setResponseHeader("X", "bar") -> inlineContent("<html><body>hello</body></html>") -> status(200) -> <shunt>
```

Comment:
Why this change?

Reply:
Because we just tell before that you should not use `Path/PathSubtree` filters in custom routes. :)

Reply:
I would keep either to leaving the `Path` predicate or changing it to the `PathSubtree`, because two of them are used in the radix tree while route matching, but `PathRegexp` is not used in this tree.
https://opensource.zalando.com/skipper/reference/architecture/#route-matching